internal/network/command_access_point_private_link_egress_endpoint_create.go

```diff
@@ -22,12 +24,17 @@ func (c *accessPointCommand) newCreateCommand() *cobra.Command {
 				Text: "Create an AWS private link egress endpoint with high availability.",
 				Code: "confluent network access-point private-link egress-endpoint create --cloud aws --gateway gw-123456 --service com.amazonaws.vpce.us-west-2.vpce-svc-00000000000000000 --high-availability",
 			},
+			examples.Example{
+				Text: "Create a named Azure private link egress endpoint.",
```

Comment:
```suggestion
				Text: `Create an Azure private link egress endpoint named "my-egress-endpoint".`,
```

Reply:
I think "Azure Private Link" is a proper noun here and should be capitalized, judging from some of the other text in this PR?

Reply:
I'll capitalize it